upgrade/1.2/scripts/sls/sls_updater.py_technical_details.md

```diff
@@ -0,0 +1,100 @@
+# `sls_updater.py` Technical Details
```

Comment:
I see that this page was based on the contents taken from the SLS Upgrade README file -- should this file be linked from that one?

Reply:
good catch.  added linking between the two pages.